tests/test_erc721.py

```diff
@@ -0,0 +1,163 @@
+# Modified from: https://github.com/ethereum/vyper/blob/25d04085a7046a97c4e4a4d3da95963e22cf9344/tests/examples/tokens/test_erc721.py
+import pytest
+
+firstTokenId = 1
+secondTokenId = 2
+unknownTokenId = 3
+ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+ERC165_INTERFACE_ID = '0x0000000000000000000000000000000000000000000000000000000001ffc9a7'
+ERC721_INTERFACE_ID = '0x0000000000000000000000000000000000000000000000000000000080ac58cd'
+
+
+@pytest.fixture
+def c(get_contract, w3):
+    with open('./erc721/ERC721.vy') as f:
+        code = f.read()
+    c = get_contract(code)
+    owner, operator, someone = w3.eth.accounts[:3]
```

Comment:
`owner` is a bit confusing with the owner of token? `minter` seems better for me.

Reply:
agree!